CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 Changes by Version
 ==================
 
-2.6.1 (unreleased)
+2.7.0 (2017-06-19)
 ------------------
 
+- Fix rate limiter balance [#135](https://github.com/uber/jaeger-client-go/pull/135) [#140](https://github.com/uber/jaeger-client-go/pull/140)
+- Default client to send Jaeger.thrift [#147](https://github.com/uber/jaeger-client-go/pull/147)
+- Rename endpoint to operation for RPC metrics [#150](https://github.com/uber/jaeger-client-go/pull/150)
```

Comment:
yeah, might be a good idea to just change the `jaeger-rpc` and `endpoint` strings back (without code changes, only constants), to maintain compatibility with Synoptic component until we do a revamp of the whole naming scheme.